src/core/server/logging/layouts/json_layout.ts

```diff
@@ -46,20 +47,28 @@ export class JsonLayout implements Layout {
 
     return {
       message: error.message,
-      name: error.name,
-      stack: error.stack,
+      type: error.name,
+      stack_trace: error.stack,
     };
   }
 
   public format(record: LogRecord): string {
-    return JSON.stringify({
-      '@timestamp': moment(record.timestamp).format('YYYY-MM-DDTHH:mm:ss.SSSZ'),
-      context: record.context,
-      error: JsonLayout.errorToSerializableObject(record.error),
-      level: record.level.id.toUpperCase(),
-      message: record.message,
-      meta: record.meta,
-      pid: record.pid,
-    });
+    return JSON.stringify(
```

Comment:
regarding this https://github.com/elastic/kibana/pull/71138#discussion_r453820515 (I don't know why I cannot comment in the thread anymore)

> merging the meta object into the record will throw a bunch of fields without formats at the logger. current behavior is json stringify, which will probably be strange for most ecs documents

We stringify `meta` fields in `pattern` layout only, which is not subject to ECS format? However, I still don't see how users can reference data from the `meta` object without using `meta.` prefix.

> For now the field usage is best effort and under migration. They'll eventually move to prefixing custom fields with es.

What is our plan for this?

> I'm leaning towards addressing 1 separately and for 2overriding before - there aren't any yet but we don't want to miss any escape or string filters.

Are you going to create a separate issue?